src/components/product/ProductGrid/ProductGrid.tsx

```diff
@@ -12,21 +13,23 @@ interface Props {
 
 function ProductGrid({ products, page, pageSize }: Props) {
   return (
-    <ul className="product-grid">
-      {products.map((product, idx) => (
-        <li key={`${product.id}`}>
-          <ProductCard
-            product={product}
-            index={pageSize * page + idx + 1}
-            bordered
-            outOfStock={
-              product.offers.offers?.[0].availability !==
-              'https://schema.org/InStock'
-            }
-          />
-        </li>
-      ))}
-    </ul>
+    <ProductGridSkeleton loading={products?.length === 0}>
```

Comment:
I think we don't need this if in here
```suggestion
    <ProductGridSkeleton loading={products.length === 0}>
```